Assets/HoloToolkit-Preview/SpectatorView/README.md

```diff
@@ -52,11 +52,41 @@ Requirements
 
 Building the SpectatorView Native Plugin
 ----------------------------------------
+The SpectatorViewPlugin will only be used in the HoloLens application. It needs to be built on a PC with visual studio.
 - See: https://github.com/Microsoft/MixedRealityToolkit/blob/master/SpectatorViewPlugin/README.md
 
- 
+After building the SpectatorViewPlugin project, you will need to copy the generated dlls into your unity project's plugins directory.
+- The compile output for SpectatorViewPlugin will likely be found at:
+>MixedRealityToolkit\SpectatorViewPlugin\SpectatorViewPlugin\Release\SpectatorViewPlugin\
+
+- The unity plugin directory will likely be found at:
+>MixedRealityToolkit-Unity\Assets\Plugins\WSA\x86\
+
+>NOTE: This unity plugin directory may not exist by default. You can create this directory in windows file explorer or in the unity editor. The Plugins directory is one of unity's special folders (https://docs.unity3d.com/Manual/SpecialFolders.html). The \WSA\x86\ subdirectories signal that these dlls are for a windows store application with x86 architecture (These are the specifications for HoloLens).
+
 Project Setup
 -------------
+### Example
+- It may prove helpful to look at an example scene before integrating SpectatorView components into your application. After having added the SpectatorViewPlugin dlls to your unity project, you should be able to build and run the provided example scene:
+    - HoloToolkit-Examples\SpectatorView\Scenes\SpectatorViewExample.unity
+
+### High Level Overview
+The logic in spectator view can be broken down into three main steps:
+
```

Comment:
Because we're adding lots of text and most people are lazy and just skim through docs, I'd recommend adding a title for each step like:
1) Device Discovery
2) Looking for Marker
3) Spatial Syncing

Also, does the device discovery still support multiple phones joining 1 HoloLens? From reading step 2) it seems like it doesn't.

Reply:
It looks like it still supports multiple hololenses. I've changed things to titles but will update the wording.

Reply:
i have made some changes to better address that you can use multiple iphones